src/js/utils/form-utils.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { Tooltip } from 'react-tippy';
 import PropTypes from 'prop-types';
-import { Translate } from 'react-localize-redux';
+import { getTranslate, Translate } from 'react-localize-redux';
 
 import 'react-tippy/dist/tippy.css';
+import connect from 'react-redux/es/connect/connect';
```

Comment:
you can use this: import { connect } from 'react-redux';
also move it before the css import (like it's done on other pages)